custom_components/luxtronik/translations/de.json

```diff
@@ -182,6 +182,24 @@
             },
             "efficiency_pump_minimal": {
                 "name": "Effizienzpumpe Minimal"
+            },
+            "cooling_threshold_temperature": {
+                "name": "T_aussen >"
```

Comment:
Can we find a better name?

Reply:
"minimale Außentemperatur für den Kühlbetrieb"?